lisa/tools/wget.py

```diff
@@ -44,6 +44,11 @@ def get(
         matched_result = self.__pattern_path.match(command_result.stdout)
         if matched_result:
             download_file_path = matched_result.group("path")
+        else:
+            raise LisaException(
+                f"Output of command (run_command): {command_result.stdout} "
+                f"doesn't match the pattern {self.__pattern_path}."
```

Comment:
It describes what happens in a technical way. I don't understand why it needs to raise an exception from the words.